Data/Sector/Hong-Kong-Sector-File.sct

```diff
@@ -893,6 +853,179 @@ WALIN N022.05.40.100 E113.58.43.200
+NIXUS N022.29.22.050 E114.10.19.700
+NOLIB N022.25.38.950 E114.00.36.840
+VH703 N022.24.23.820 E113.57.20.920
+VH704 N022.21.42.460 E113.50.21.180
+VH715 N022.18.18.690 E113.47.11.730
+VH716 N022.16.23.230 E113.49.35.170
+VH717 N022.16.42.960 E113.50.36.280
+LIKBI N022.24.43.380 E114.15.32.100
+HH813 N022.22.49.720 E114.09.36.810
+RW25L N022.18.26.750 E113.55.58.150
+HH841 N022.17.08.540 E113.51.55.540
+HH846 N022.15.10.930 E113.48.48.940
+HH848 N022.11.48.890 E113.48.41.360
+HH849 N022.09.49.330 E113.51.03.910
+HH844 N022.17.16.040 E114.16.15.090
+HH845 N022.18.42.080 E114.21.08.000
+HH814 N022.22.58.610 E114.10.04.780
+HH815 N022.23.09.080 E114.10.37.300
+HH861 N022.17.16.920 E113.52.21.600
+HH862 N022.16.11.530 E113.50.25.470
+HH865 N022.09.53.610 E113.54.49.530
+LUDLA N022.13.40.790 E114.04.28.750
+HH982 N022.12.30.910 E113.54.08.120
+HH985 N022.13.24.620 E113.53.13.940
+HH871 N022.17.03.650 E114.05.40.960
+HH872 N022.18.24.890 E114.05.46.320
+HH873 N022.20.24.090 E114.02.02.860
+HH874 N022.19.51.370 E114.00.21.120
+HH981 N022.17.53.680 E114.02.58.660
+TONIC N022.12.47.880 E113.43.21.850
+VH711 N022.17.04.628 E113.48.59.260
+RW07C N022.18.40.580 E113.53.56.640
+HH747 N022.20.00.420 E113.58.04.560
+HH748 N022.27.40.860 E114.05.14.120
+HH741 N022.29.58.660 E114.11.14.890
+HH742 N022.18.46.900 E114.21.54.810
+HH743 N022.09.58.850 E114.13.06.230
+HH744 N022.01.23.710 E113.56.19.340
+VH710 N022.15.58.800 E113.45.35.610
+HH730 N022.17.02.670 E113.48.53.160
+GONAT N022.32.32.300 E114.15.27.060
+HH827 N022.28.39.550 E114.17.09.590
+CANUP N022.25.02.540 E114.13.46.400
+HH832 N022.23.36.120 E114.09.16.390
+RW25C N022.19.12.850 E113.55.36.780
+HH821 N022.17.54.880 E113.51.34.920
+HH822 N022.21.58.900 E113.49.43.760
+VH523 N022.24.20.510 E113.53.58.890
+VH526 N022.29.52.410 E114.08.27.150
+HH834 N022.23.44.660 E114.09.43.130
+HH835 N022.23.55.530 E114.10.16.990
+LAMMA N022.16.55.610 E114.05.10.550
+VH533 N022.19.53.470 E114.04.36.720
+HH851 N022.21.02.310 E114.01.17.040
+HH852 N022.20.34.770 E113.59.51.340
+HH867 N022.18.27.110 E113.53.14.840
+VH525 N022.23.15.380 E113.51.16.250
+VH524 N022.25.56.630 E113.58.09.870
+HH987 N022.20.53.000 E113.52.20.490
+TUTBA N022.14.01.820 E113.42.44.920
+VH701 N022.16.50.880 E113.45.21.490
+VH702 N022.17.50.070 E113.48.24.640
+VH726 N022.20.36.290 E113.57.00.200
+VH727 N022.26.18.620 E114.01.39.190
+VH708 N022.17.46.180 E113.48.12.620
+RW07L N022.19.17.720 E113.52.56.260
+TOPUN N022.22.54.170 E114.04.09.150
+VH536 N022.21.51.550 E114.00.54.180
+VH528 N022.21.31.880 E113.59.52.970
+RW25R N022.19.54.450 E113.54.50.240
+VH522 N022.19.17.080 E113.52.54.280
+BOKAG N022.31.46.650 E114.13.27.060
+HH806 N022.28.18.610 E114.14.58.760
+RUNSU N022.25.22.960 E114.11.53.570
+VH515 N022.24.53.870 E114.10.22.720
+SAGNI N022.23.46.630 E114.06.52.680
+VH510 N022.23.44.370 E114.06.45.650
+VH505 N022.24.24.160 E114.08.49.880
+
+;3RS Enroute
+NUDPI N021.54.19.000 E115.37.28.000
+MEBKI N021.44.33.000 E115.34.54.000
+
```

Comment:
Would you mind sorting the fixes into the original alphabetical list

Reply:
```ini
;3RS Terminal
ABBEY N022.16.11.000 E114.55.25.920
ADLAD N022.30.40.740 E114.19.52.570
AGOMU N022.30.15.170 E114.22.29.860
ANRUX N022.27.05.730 E114.12.41.760
ARGON N022.11.52.560 E113.43.47.670
BETTY N021.29.10.820 E114.33.31.920
BOKAG N022.31.46.650 E114.13.27.060
BORDA N021.41.32.370 E113.24.29.010
CANTO N021.39.02.630 E113.42.25.090
CANUP N022.25.02.540 E114.13.46.400
DAKIG N021.50.52.900 E114.26.33.330
DALOL N021.44.36.900 E114.50.45.340
DUMEP N021.44.36.300 E115.12.49.870
GONAT N022.32.32.300 E114.15.27.060
GUAVA N022.09.36.100 E114.03.36.300
HH301 N022.19.29.010 E113.59.11.610
HH302 N022.20.00.430 E113.58.04.560
HH303 N022.20.50.020 E113.57.42.830
HH311 N022.14.52.340 E114.05.12.730
HH322 N022.15.08.880 E114.05.12.880
HH332 N022.15.22.170 E114.05.13.000
HH341 N022.19.25.150 E114.04.19.860
HH342 N022.17.12.510 E114.05.14.010
HH355 N022.19.45.970 E114.04.15.230
HH356 N022.17.22.570 E114.05.14.100
HH363 N022.19.29.680 E114.04.09.360
HH364 N022.16.56.520 E114.05.13.860
HH381 N022.09.02.100 E114.15.06.660
HH382 N022.11.23.060 E114.26.04.390
HH384 N022.23.11.510 E114.21.17.830
HH393 N022.19.43.830 E114.16.15.260
HH411 N022.17.08.520 E113.51.55.320
HH412 N022.15.10.920 E113.48.48.950
HH413 N022.11.48.900 E113.48.41.370
HH414 N022.09.44.510 E113.51.09.660
HH421 N022.15.31.130 E113.44.09.410
HH422 N022.12.58.530 E113.42.27.410
HH423 N022.07.26.100 E113.44.59.400
HH471 N022.07.28.610 E114.03.33.280
HH472 N022.05.51.490 E114.13.05.410
HH477 N021.57.29.800 E114.12.54.070
HH478 N021.52.29.090 E114.12.47.310
HH481 N022.11.23.110 E114.01.29.600
HH482 N022.17.15.280 E114.07.22.590
HH484 N022.02.23.210 E113.56.01.920
HH485 N022.04.19.900 E114.00.35.870
HH486 N022.06.56.320 E114.06.43.850
HH487 N022.15.17.490 E114.12.46.600
HH511 N021.53.32.240 E114.11.31.990
HH512 N022.01.21.800 E114.31.56.700
HH622 N021.57.14.020 E114.26.43.790
HH623 N021.51.34.240 E114.12.32.620
HH730 N022.17.02.670 E113.48.53.160
HH741 N022.29.58.660 E114.11.14.890
HH742 N022.18.46.900 E114.21.54.810
HH743 N022.09.58.850 E114.13.06.230
HH744 N022.01.23.710 E113.56.19.340
HH747 N022.20.00.420 E113.58.04.560
HH748 N022.27.40.860 E114.05.14.120
HH806 N022.28.18.610 E114.14.58.760
HH813 N022.22.49.720 E114.09.36.810
HH814 N022.22.58.610 E114.10.04.780
HH815 N022.23.09.080 E114.10.37.300
HH821 N022.17.54.880 E113.51.34.920
HH822 N022.21.58.900 E113.49.43.760
HH827 N022.28.39.550 E114.17.09.590
HH832 N022.23.36.120 E114.09.16.390
HH834 N022.23.44.660 E114.09.43.130
HH835 N022.23.55.530 E114.10.16.990
HH841 N022.17.08.540 E113.51.55.540
HH844 N022.17.16.040 E114.16.15.090
HH845 N022.18.42.080 E114.21.08.000
HH846 N022.15.10.930 E113.48.48.940
HH848 N022.11.48.890 E113.48.41.360
HH849 N022.09.49.330 E113.51.03.910
HH851 N022.21.02.310 E114.01.17.040
HH852 N022.20.34.770 E113.59.51.340
HH861 N022.17.16.920 E113.52.21.600
HH862 N022.16.11.530 E113.50.25.470
HH865 N022.09.53.610 E113.54.49.530
HH867 N022.18.27.110 E113.53.14.840
HH871 N022.17.03.650 E114.05.40.960
HH872 N022.18.24.890 E114.05.46.320
HH873 N022.20.24.090 E114.02.02.860
HH874 N022.19.51.370 E114.00.21.120
HH941 N022.17.37.960 E114.02.12.360
HH942 N022.17.14.130 E114.01.44.030
HH955 N022.17.58.850 E114.02.08.750
HH956 N022.17.24.310 E114.01.27.970
HH963 N022.17.04.890 E114.01.24.200
HH964 N022.16.58.350 E114.01.16.750
HH981 N022.17.53.680 E114.02.58.660
HH982 N022.12.30.910 E113.54.08.120
HH985 N022.13.24.620 E113.53.13.940
HH987 N022.20.53.000 E113.52.20.490
LAMMA N022.16.55.610 E114.05.10.550
LARIT N021.53.41.370 E115.16.06.560
LEKEN N021.53.01.060 E114.54.44.950
LIKBI N022.24.43.380 E114.15.32.100
LIMES N022.06.25.600 E113.46.32.600
LUBMO N021.57.55.240 E114.31.52.980
LUDLA N022.13.40.790 E114.04.28.750
MANGO N021.36.49.000 E114.30.53.000
MURRY N021.45.28.370 E113.48.41.970
MUSEL N022.15.53.990 E114.46.52.130
NIXUS N022.29.22.050 E114.10.19.700
NOLIB N022.25.38.950 E114.00.36.840
PECAN N021.26.20.190 E114.02.05.640
PORPA N022.20.09.100 E114.01.16.300
PRAWN N022.16.05.400 E113.48.40.100
RAMEN N022.09.39.220 E114.05.09.890
RIVMI N022.24.03.960 E114.19.09.280
ROCCA N021.31.55.420 E113.34.51.440
ROVER N022.20.35.580 E114.01.39.120
RUMSY N022.04.56.940 E113.48.16.790
RUNSU N022.25.22.960 E114.11.53.570
RW07C N022.18.40.580 E113.53.56.640
RW07L N022.19.17.720 E113.52.56.260
RW07R N022.17.48.030 E113.53.57.990
RW25C N022.19.12.850 E113.55.36.780
RW25L N022.18.26.750 E113.55.58.150
RW25R N022.19.54.450 E113.54.50.240
SAGNI N022.23.46.630 E114.06.52.680
SAMED N022.04.43.000 E114.26.31.830
SEGPO N021.57.32.770 E114.21.18.590
SILVA N021.51.04.500 E113.54.10.880
SOKOE N022.04.41.200 E113.50.38.100
SOSLU N021.38.44.030 E114.05.18.390
STELA N022.11.53.290 E113.43.49.110
SUDVA N022.04.25.140 E114.21.33.870
SUGAP N021.55.57.440 E114.02.01.640
SUPAD N022.00.05.990 E114.07.30.720
TEDUR N022.11.18.770 E114.23.51.260
TEGUB N022.24.18.280 E114.04.54.690
TONIC N022.12.47.880 E113.43.21.850
TOPUN N022.22.54.170 E114.04.09.150
TUTBA N022.14.01.820 E113.42.44.920
VANPU N022.03.57.950 E113.48.03.850
VEDMU N022.08.14.020 E113.55.08.180
VEDVO N022.04.38.190 E114.15.43.050
VEMSO N022.14.25.060 E114.18.52.280
VH505 N022.24.24.160 E114.08.49.880
VH510 N022.23.44.370 E114.06.45.650
VH515 N022.24.53.870 E114.10.22.720
VH522 N022.19.17.080 E113.52.54.280
VH523 N022.24.20.510 E113.53.58.890
VH524 N022.25.56.630 E113.58.09.870
VH525 N022.23.15.380 E113.51.16.250
VH526 N022.29.52.410 E114.08.27.150
VH528 N022.21.31.880 E113.59.52.970
VH533 N022.19.53.470 E114.04.36.720
VH536 N022.21.51.550 E114.00.54.180
VH701 N022.16.50.880 E113.45.21.490
VH702 N022.17.50.070 E113.48.24.640
VH703 N022.24.23.820 E113.57.20.920
VH704 N022.21.42.460 E113.50.21.180
VH708 N022.17.46.180 E113.48.12.620
VH710 N022.15.58.800 E113.45.35.610
VH711 N022.17.04.628 E113.48.59.260
VH715 N022.18.18.690 E113.47.11.730
VH716 N022.16.23.230 E113.49.35.170
VH717 N022.16.42.960 E113.50.36.280
VH720 N022.15.05.940 E113.45.39.560
VH721 N022.16.12.680 E113.49.04.870
VH726 N022.20.36.290 E113.57.00.200
VH727 N022.26.18.620 E114.01.39.190
XEMEK N021.38.16.170 E114.59.43.040

;3RS Enroute
MEBKI N021.44.33.000 E115.34.54.000
NUDPI N021.54.19.000 E115.37.28.000
```